aws/data_source_aws_subnet_test.go

```diff
@@ -134,6 +147,45 @@ func TestAccDataSourceAwsSubnet_ipv6ByIpv6CidrBlock(t *testing.T) {
 	})
 }
 
+func TestAccDataSourceAwsSubnet_CustomerOwnedIpv4Pool(t *testing.T) {
+	// Hide Outposts testing behind consistent environment variable
+	outpostArn := os.Getenv("AWS_OUTPOST_ARN")
```

Comment:
Is there any advantage in moving this (and the subsequent `AWS_COIP_POOL_ID`) into `PreCheck` functions?

Reply:
Indeed 👍  Fetching the value via environment variable will likely be replaced by an upcoming `aws_outposts_outpost` data source, however, we will still want a way to `PreCheck` that an Outpost is available.